eth_account/account.py

```diff
@@ -392,7 +392,6 @@ def signTransaction(self, transaction_dict, private_key):
         .. code-block:: python
 
             >>> transaction = {
-                    # Note that the address must be in checksum format:
```

Comment:
Let's leave this message in, but add `or native bytes`. It's nice to hint to people who try to use a non-checksum address and get failures.

Reply:
Fixed!

https://github.com/ethereum/eth-account/pull/36/commits/abbae838ccb5297469e4df5fc266e575649939f6